src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs

```diff
@@ -31,7 +31,8 @@ public NamespaceFixup()
             }
 
             // Microsoft.CodeDom
-            var mscodedomTypes = TypeLoader.LoadTypes(typeof(Microsoft.CodeDom.CodeObject).GetTypeInfo().Assembly, Verbosity.Silent);
+            var mscodedomTypes = TypeLoader.LoadTypes(typeof(Microsoft.CodeDom.CodeObject).GetTypeInfo().Assembly, Verbosity.Silent, null, null);
```

Comment:
same here?

Reply:
Yes, I added two parameters in LoadTypes method